### ISSUE TITLE
Missing Abstracts

### ISSUE THREAD

**Issue by cpetrule** [ropensci/openalexR#80]
The `ab` field  used to contain the abstract of the publication. For some reason now it is all NAs. Is it a deprecated field? Here a comparison of metadata retrieved some months ago vs today (after updating to the `ropensci `instead of `massimoaria `version of `openalexR`).
![Screenshot 2023-02-28 161228](https://user-images.githubusercontent.com/105730111/221895959-993f8804-bae9-4682-82b4-2172b3272133.png)
![Screenshot 2023-02-28 161249](https://user-images.githubusercontent.com/105730111/221895966-17c3f003-b1bb-496f-a95e-b2937212836c.png)

**Comment by trangdata** [comment]
Thank you for reporting @cpetrule! And yes, we should have made this clearer: as of a v1.0.0, the argument `abstract` in `oa_fetch` defaults to [FALSE](https://github.com/ropensci/openalexR/blob/57b20b0fbaa7ace61466f19d281ec996fceb9562/R/oa_fetch.R#L65) to save compute/query time/space. You will need to set `abstract = TRUE` to retrieve abstracts for each article.

If that still doesn't work, could you share a reproducible example, please? 🙏🏽

**Comment by cpetrule** [comment]
it works! 🙌 are the new changes documented?

**Comment by trangdata** [comment]
@cpetrule No... We need to be better at getting more specific in [NEWS](https://ropensci.github.io/openalexR/news/index.html). Thank you for the suggestion. I documented this change in 0585062a5d196678827705c898dbfd78c39f7938.

**Comment by agricolamz** [comment]
Oh, that was cruel... Good that I see this issue, thx @cpetrule!
